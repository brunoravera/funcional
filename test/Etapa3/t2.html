<html>
	<head>
		<link rel="stylesheet" href="gift.css">
	</head>
	<body>
<div class="giftComentario">
  ejemplo  de  verdadero falso
</div>
<span class="giftNombre">
fv.1</span>
<div class="giftFragmento">
Grant murio en 1886</div>


<span class="giftFV">
False</span>

<div class="giftFragmento">
</div>


<div class="giftComentario">
 fin de  pregunta
</div>

<div class="giftComentario">
  ejemplo  de  multiple  opcion
</div>
<span class="giftNombre">
mo.1</span>
<div class="giftFragmento">
</div>


<div class="giftMO">
\item <div class="giftFragmento">
Grant</div>


\item <div class="giftFragmento">
el gato de  Grant</div>


\item <div class="giftFragmento">
Tumbledore</div>


</div>

<div class="giftFragmento">
esta sepultado  en la
tumba de  Grant.</div>


<div class="giftComentario">
 fin de  pregunta
</div>

<div class="giftComentario">
  ejemplo  de  multiple  opcion
</div>
<span class="giftNombre">
mo.2</span>
<div class="giftFragmento">
Indique quien esta sepultado  en la  tumba de  Grant.</div>


<div class="giftMO">
\item <div class="giftFragmento">
Grant</div>


\item <div class="giftFragmento">
el gato de Grant</div>


\item <div class="giftFragmento">
Tumbledore</div>


</div>

<div class="giftFragmento">
</div>


<div class="giftComentario">
 fin de  pregunta
</div>

<div class="giftComentario">
  ejemplo  de  desarrollo
</div>
<span class="giftNombre">
 desarrollo .1</span>
<div class="giftFragmento">
Cuente  acerca  de las  becas  que recibio Grant  como  estudiante.</div>


<span class="giftEssay">
</span>

<div class="giftFragmento">
</div>


<div class="giftComentario">
 fin de  pregunta
</div>

<div class="giftComentario">
  ejemplo  de  respuesta  corta
</div>
<span class="giftNombre">
 corta .1</span>
<div class="giftFragmento">
Quien esta sepultado  en la  tumba de  Grant ?</div>


<div class="giftShort">
\item <div class="giftFragmento">
Grant</div>


\item <div class="giftFragmento">
Ulysses
S. Grant</div>


\item <div class="giftFragmento">
Ulysses  Grant</div>


</div>

<div class="giftFragmento">
</div>


<div class="giftComentario">
 fin de  pregunta
</div>

<div class="giftComentario">
  ejemplo  de  respuesta  corta
</div>
<span class="giftNombre">
 corta .2</span>
<div class="giftFragmento">
</div>


<div class="giftShort">
\item <div class="giftFragmento">
Grant</div>


\item <div class="giftFragmento">
Ulysses S. Grant</div>


\item <div class="giftFragmento">
Ulysses  Grant</div>


</div>

<div class="giftFragmento">
esta sepultado  en
la  tumba de  Grant.</div>


<div class="giftComentario">
 fin de  pregunta
</div>

<div class="giftComentario">

</div>
<span class="giftNombre">
 ejemplo  del  enunciado </span>
<div class="giftFragmento">
Complete  el  programa</div>

<span class="giftCode">
foo x = "Si un  quilo de  papas
cuesta $" ++ x ++ " entonces  dos  quilos  cuestan" ++ ...</span>

<div class="giftFragmento">
</div>


<span class="giftEssay">
</span>

<div class="giftFragmento">
.
Cual es el tipo de</div>

<span class="giftCode">
foo</span>

<div class="giftFragmento">
?</div>


<div class="giftMO">
\item <div class="giftFragmento">
String  -> String</div>


\item <span class="giftCode">
Papa  -> Papa  -> Papa</span>

<div class="giftFragmento">
</div>


\item <span class="giftCode">
$ -> $</span>

<div class="giftFragmento">
</div>


\item <span class="giftMATH">
-></span>

<div class="giftFragmento">
</div>


</div>

<div class="giftFragmento">
</div>


<div class="giftComentario">

</div>

	</body>
</html>
