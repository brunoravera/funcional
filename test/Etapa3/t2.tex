\documentclass{gift}
\begin{document}
\begin{giftComentario}
  ejemplo  de  verdadero falso
\end  {giftComentario}
\giftNombre{fv.1}
\begin{giftFragmento}
Grant murio en 1886\end  {giftFragmento}


\giftFV{False}

\begin{giftFragmento}
\end  {giftFragmento}


\begin{giftComentario}
 fin de  pregunta
\end  {giftComentario}

\begin{giftComentario}
  ejemplo  de  multiple  opcion
\end  {giftComentario}
\giftNombre{mo.1}
\begin{giftFragmento}
\end  {giftFragmento}


\begin{giftMO}
\item \begin{giftFragmento}
Grant\end  {giftFragmento}


\item \begin{giftFragmento}
el gato de  Grant\end  {giftFragmento}


\item \begin{giftFragmento}
Tumbledore\end  {giftFragmento}


\end  {giftMO}

\begin{giftFragmento}
esta sepultado  en la
tumba de  Grant.\end  {giftFragmento}


\begin{giftComentario}
 fin de  pregunta
\end  {giftComentario}

\begin{giftComentario}
  ejemplo  de  multiple  opcion
\end  {giftComentario}
\giftNombre{mo.2}
\begin{giftFragmento}
Indique quien esta sepultado  en la  tumba de  Grant.\end  {giftFragmento}


\begin{giftMO}
\item \begin{giftFragmento}
Grant\end  {giftFragmento}


\item \begin{giftFragmento}
el gato de Grant\end  {giftFragmento}


\item \begin{giftFragmento}
Tumbledore\end  {giftFragmento}


\end  {giftMO}

\begin{giftFragmento}
\end  {giftFragmento}


\begin{giftComentario}
 fin de  pregunta
\end  {giftComentario}

\begin{giftComentario}
  ejemplo  de  desarrollo
\end  {giftComentario}
\giftNombre{ desarrollo .1}
\begin{giftFragmento}
Cuente  acerca  de las  becas  que recibio Grant  como  estudiante.\end  {giftFragmento}


\giftEssay{}

\begin{giftFragmento}
\end  {giftFragmento}


\begin{giftComentario}
 fin de  pregunta
\end  {giftComentario}

\begin{giftComentario}
  ejemplo  de  respuesta  corta
\end  {giftComentario}
\giftNombre{ corta .1}
\begin{giftFragmento}
Quien esta sepultado  en la  tumba de  Grant ?\end  {giftFragmento}


\begin{giftShort}
\item \begin{giftFragmento}
Grant\end  {giftFragmento}


\item \begin{giftFragmento}
Ulysses
S. Grant\end  {giftFragmento}


\item \begin{giftFragmento}
Ulysses  Grant\end  {giftFragmento}


\end  {giftShort}

\begin{giftFragmento}
\end  {giftFragmento}


\begin{giftComentario}
 fin de  pregunta
\end  {giftComentario}

\begin{giftComentario}
  ejemplo  de  respuesta  corta
\end  {giftComentario}
\giftNombre{ corta .2}
\begin{giftFragmento}
\end  {giftFragmento}


\begin{giftShort}
\item \begin{giftFragmento}
Grant\end  {giftFragmento}


\item \begin{giftFragmento}
Ulysses S. Grant\end  {giftFragmento}


\item \begin{giftFragmento}
Ulysses  Grant\end  {giftFragmento}


\end  {giftShort}

\begin{giftFragmento}
esta sepultado  en
la  tumba de  Grant.\end  {giftFragmento}


\begin{giftComentario}
 fin de  pregunta
\end  {giftComentario}

\begin{giftComentario}

\end  {giftComentario}
\giftNombre{ ejemplo  del  enunciado }
\begin{giftFragmento}
Complete  el  programa\end  {giftFragmento}

\verb!foo x = "Si un  quilo de  papas
cuesta $" ++ x ++ " entonces  dos  quilos  cuestan" ++ ...!
\begin{giftFragmento}
\end  {giftFragmento}


\giftEssay{}

\begin{giftFragmento}
.
Cual es el tipo de\end  {giftFragmento}

\verb!foo!
\begin{giftFragmento}
?\end  {giftFragmento}


\begin{giftMO}
\item \begin{giftFragmento}
String  -> String\end  {giftFragmento}


\item \verb!Papa  -> Papa  -> Papa!
\begin{giftFragmento}
\end  {giftFragmento}


\item \verb!$ -> $!
\begin{giftFragmento}
\end  {giftFragmento}


\item $->$
\begin{giftFragmento}
\end  {giftFragmento}


\end  {giftMO}

\begin{giftFragmento}
\end  {giftFragmento}


\begin{giftComentario}

\end  {giftComentario}

\end{document}
