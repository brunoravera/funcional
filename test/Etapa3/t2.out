CUESTIONARIO
============

PREGUNTA fv.1
COMENTARIO INICIAL:   ejemplo  de  verdadero falso
[Q [TXT "Grant murio en 1886"],A (FV False),Q [TXT ""]]
COMENTARIO FINAL:  fin de  pregunta
------------
PREGUNTA mo.1
COMENTARIO INICIAL:   ejemplo  de  multiple  opcion
[Q [TXT ""],A (MO [OK [TXT "Grant"],NOK [TXT "el gato de  Grant"],NOK [TXT "Tumbledore"]]),Q [TXT "esta sepultado  en la\ntumba de  Grant."]]
COMENTARIO FINAL:  fin de  pregunta
------------
PREGUNTA mo.2
COMENTARIO INICIAL:   ejemplo  de  multiple  opcion
[Q [TXT "Indique quien esta sepultado  en la  tumba de  Grant."],A (MO [OK [TXT "Grant"],NOK [TXT "el gato de Grant"],NOK [TXT "Tumbledore"]]),Q [TXT ""]]
COMENTARIO FINAL:  fin de  pregunta
------------
PREGUNTA  desarrollo .1
COMENTARIO INICIAL:   ejemplo  de  desarrollo
[Q [TXT "Cuente  acerca  de las  becas  que recibio Grant  como  estudiante."],A ESSAY,Q [TXT ""]]
COMENTARIO FINAL:  fin de  pregunta
------------
PREGUNTA  corta .1
COMENTARIO INICIAL:   ejemplo  de  respuesta  corta
[Q [TXT "Quien esta sepultado  en la  tumba de  Grant ?"],A (SHORT [OK [TXT "Grant"],OK [TXT "Ulysses\nS. Grant"],OK [TXT "Ulysses  Grant"]]),Q [TXT ""]]
COMENTARIO FINAL:  fin de  pregunta
------------
PREGUNTA  corta .2
COMENTARIO INICIAL:   ejemplo  de  respuesta  corta
[Q [TXT ""],A (SHORT [OK [TXT "Grant"],OK [TXT "Ulysses S. Grant"],OK [TXT "Ulysses  Grant"]]),Q [TXT "esta sepultado  en\nla  tumba de  Grant."]]
COMENTARIO FINAL:  fin de  pregunta
------------
PREGUNTA  ejemplo  del  enunciado 
COMENTARIO INICIAL: 
[Q [TXT "Complete  el  programa",CODE "foo x = \"Si un  quilo de  papas\ncuesta $\" ++ x ++ \" entonces  dos  quilos  cuestan\" ++ ...",TXT ""],A ESSAY,Q [TXT ".\nCual es el tipo de",CODE "foo",TXT "?"],A (MO [OK [TXT "String  -> String"],NOK [CODE "Papa  -> Papa  -> Papa",TXT ""],NOK [CODE "$ -> $",TXT ""],NOK [MATH "->",TXT ""]]),Q [TXT ""]]
COMENTARIO FINAL: 
------------
